<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binomialrechner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/1998611.png</normaloff>../../../Downloads/1998611.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0">
       <widget class="QLineEdit" name="underEdit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="upperEdit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="underLabel">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Untere Grenze</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="_n_label">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Stichprobenumfang</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="upperLabel">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Obere Grenze</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="_sigma_label">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Standardabweichung</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="_n_edit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>n</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="_p_edit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kommazahlen nach englischem Vorbild eingeben (mit . statt , )&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="_mu_label">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Erwartungswert</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="_m_edit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>µ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="_p_label">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Trefferwahrscheinlichkeit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="_sigma_edit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>σ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="showTable">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>not programmed yet</string>
        </property>
        <property name="whatsThis">
         <string>not programmed yet</string>
        </property>
        <property name="text">
         <string>Tabelle zeigen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="calWithParam">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mit Erwartungswert und Standardabweichung rechnen</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cumulativeP">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>kumulierte Wahrscheinlichkeitsrechnung</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="showHisto">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>not programmed yet</string>
        </property>
        <property name="whatsThis">
         <string>not programmed yet</string>
        </property>
        <property name="text">
         <string>Histogramm zeigen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="hiddenOutput">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Rechner</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pOutput">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Binomial</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionreadme">
   <property name="text">
    <string>readme</string>
   </property>
  </action>
  <action name="actionswitch_to_English">
   <property name="text">
    <string>switch to English</string>
   </property>
  </action>
  <action name="actionreset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
